cmd/agent/daemon/state/netflow_pipeline.go

```diff
@@ -55,9 +55,7 @@ func (c *Controller) runNetflowPipeline(ctx context.Context) error {
 	c.log.Info("running netflow pipeline")
 	defer c.log.Info("netflow pipeline done")
 
-	kubeCtx, cancel := context.WithTimeout(ctx, 20*time.Second)
-	defer cancel()
-	clusterInfo, err := c.getClusterInfo(kubeCtx)
+	clusterInfo, err := c.getClusterInfo(ctx)
```

Comment:
Why was the timeout removed?

Reply:
Before if controller would not start in 20 seconds netflow pipeline would continue running without cluster info and will never enrich k8s destinations. 

Ideally these should be some periodic loop with the refresh, but this is a bigger change. I can revert this and create separate PR, since this is more of a bug right now.

Reply:
Would be nice if you could add a TODO comment.